packages/augur-ui/src/modules/account/components/notifications/notifications.tsx

```diff
@@ -25,28 +25,15 @@ import {
   UnsignedOrdersTemplate,
   ProceedsToClaimTemplate,
   ProceedsToClaimOnHoldTemplate,
-  OrphanOrdersTemplate
 } from "modules/account/components/notifications/notifications-templates";
 
 import * as constants from "modules/common-elements/constants";
+import { Notification } from "modules/types";
 
-export interface INotifications {
-  id: string;
-  type: string;
-  isImportant: boolean;
-  isNew: boolean;
-  title: string;
-  buttonLabel: string;
-  buttonAction: Function;
-  Template: ReactNode;
-  market: Market | null;
-  markets: Array<string>;
-  claimReportingFees?: Object;
-  totalProceeds?: number;
-}
+const { NOTIFICATION_TYPES } = constants;
```

Comment:
let's not double up on constants, just deconstruct on the import:

`import { NOTIFICATION_TYPES } from "modules/common-elements/constants";`

Reply:
I think this is the way to go. NOTIFICATION_TYPES is referred to 18 times in this file. 
other way to go would be `const NOTIFICATION_TYPES = constants.NOTIFICATION_TYPES;`

What do you think?

Reply:
merged from master, this is no longer an issue.